src/queries/eventRecords.js

```diff
@@ -138,7 +114,44 @@ export const GET_EVENT_RECORDS = gql`
   }
 
   fragment defaultFields on EventRecord {
-    ${defaultFields}
+    ${defaultFragment}
+  }
+
+  fragment dateFields on EventRecord {
+    ${dateFragment}
+  }
+`;
+
+export const GET_EVENT_RECORDS_COUNT = gql`
+  query EventRecords(
+    $ids: [ID]
+    $limit: Int
+    $take: Int
+    $location: String
+    $locations: [String]
+    $offset: Int
+    $order: EventRecordsOrder
+    $categoryId: ID
+    $dateRange: [String]
+    $dataProvider: String
+    $dataProviderId: ID
+  ) {
+    eventRecords(
+      ids: $ids
+      limit: $limit
+      take: $take
+      location: $location
+      locations: $locations
+      skip: $offset
+      order: $order
+      categoryId: $categoryId
+      dateRange: $dateRange
+      dataProvider: $dataProvider
+      dataProviderId: $dataProviderId
+    ) {
+      id
+      ...dateFields
```

Comment:
Do we need `dateFields`? since this query only shows the count of current events, I think the `id` value is enough.

Reply:
Ach okay. I didn't think about the server side. 👍